testing/scenario_app/bin/android_integration_tests.dart

```diff
@@ -16,8 +16,44 @@ import 'utils/logs.dart';
 import 'utils/process_manager_extension.dart';
 import 'utils/screenshot_transformer.dart';
 
-const int tcpPort = 3001;
-
+/// Runs the Android integration tests on a connected device or emulator.
+///
+/// The tests are uploaded and run on the device using `adb`, and screenshots
+/// are captured and compared using Skia Gold (if available, for example on CI).
+///
+/// ## Usage
+///
+/// ```sh
+/// dart bin/android_integration_tests.dart \
+///   --adb ../third_party/android_tools/sdk/platform-tools/adb \
+///   --out-dir ../out/android_debug_unopt_arm64
+/// ```
+///
```

Comment:
Maybe this should be in the README.md instead of in the source file in order to minimize where people look for information.

Reply:
Done